std/machines/hash/keccakf16.asm

```diff
@@ -0,0 +1,576 @@
+
+    col witness operation_id;
+
+    let NUM_ROUNDS: int = 24;
+
+    // pub struct KeccakCols<T> {
```

Comment:
Think these comments are nice, but maybe mention a link to the source code with some acknowledgement to Plonky3 then?

Reply:
Acknowledgement added.